.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+*Issue #, if available:*
```

Comment:
IonJs uses formated commit messages to automate the release process and versioning. 
We'll have to see how to incorporate this template. 

See https://github.com/amzn/ion-js/blob/master/CONTRIBUTE.md#pull-requests